osu.Game.Rulesets.Mania/Skinning/Argon/ArgonHitExplosion.cs

```diff
@@ -78,16 +76,8 @@ private void load(IScrollingInfo scrollingInfo)
 
         private void onDirectionChanged(ValueChangedEvent<ScrollingDirection> direction)
         {
-            if (direction.NewValue == ScrollingDirection.Up)
-            {
-                Anchor = Anchor.TopCentre;
-                Y = ArgonNotePiece.NOTE_HEIGHT / 2;
-            }
-            else
-            {
-                Anchor = Anchor.BottomCentre;
-                Y = -ArgonNotePiece.NOTE_HEIGHT / 2;
-            }
+            Anchor = largeFaint.Anchor = largeFaint.Origin = direction.NewValue == ScrollingDirection.Up ? Anchor.TopCentre : Anchor.BottomCentre;
```

Comment:
Personally I find the old code 100% more legible. Any reason for changing the code style?

Reply:
I didn't like that with the previous code I had to repeat the compound assignment. Seemed error prone, but I'll revert to the old style if you think that's best.

Reply:
You mean the assignment to the three different drawables? I actually didn't even see that when reading this due to the single-line formatting so I think that's another good reason to split things out 😅.

I'd even be okay with

```csharp
diff --git a/osu.Game.Rulesets.Mania/Skinning/Argon/ArgonHitExplosion.cs b/osu.Game.Rulesets.Mania/Skinning/Argon/ArgonHitExplosion.cs
index fb38b536dc..d490d3f944 100644
--- a/osu.Game.Rulesets.Mania/Skinning/Argon/ArgonHitExplosion.cs
+++ b/osu.Game.Rulesets.Mania/Skinning/Argon/ArgonHitExplosion.cs
@@ -76,8 +76,20 @@ private void load(IScrollingInfo scrollingInfo)
 
         private void onDirectionChanged(ValueChangedEvent<ScrollingDirection> direction)
         {
-            Anchor = largeFaint.Anchor = largeFaint.Origin = direction.NewValue == ScrollingDirection.Up ? Anchor.TopCentre : Anchor.BottomCentre;
-            Y = direction.NewValue == ScrollingDirection.Up ? ArgonNotePiece.NOTE_HEIGHT / 2 : -ArgonNotePiece.NOTE_HEIGHT / 2;
+            if (direction.NewValue == ScrollingDirection.Up)
+            {
+                Anchor = Anchor.TopCentre;
+                largeFaint.Anchor = Anchor.TopCentre;
+                largeFaint.Origin = Anchor.TopCentre;
+                Y = ArgonNotePiece.NOTE_HEIGHT / 2;
+            }
+            else
+            {
+                Anchor = Anchor.BottomCentre;
+                largeFaint.Anchor = Anchor.BottomCentre;
+                largeFaint.Origin = Anchor.BottomCentre;
+                Y = -ArgonNotePiece.NOTE_HEIGHT / 2;
+            }
         }
 
         public void Animate(JudgementResult result)

```

Reply:
Sure, I can apply as proposed instead.